helm_deploy/hmpps-prisoner-from-nomis-migration/values.yaml

```diff
@@ -84,6 +84,10 @@ generic-service:
       HMPPS_SQS_QUEUES_EVENTSENTENCING_QUEUE_NAME: "sqs_name"
     prisoner-from-nomis-sentencing-dl-queue:
       HMPPS_SQS_QUEUES_EVENTSENTENCING_DLQ_NAME: "sqs_name"
+    prisoner-from-nomis-non-associations-queue:
+      HMPPS_SQS_QUEUES_EVENTNON_ASSOCIATIONS_QUEUE_NAME: "sqs_name"
+    prisoner-from-nomis-non-associations-dl-queue:
+      HMPPS_SQS_QUEUES_EVENTNON_ASSOCIATIONS_DLQ_NAME: "sqs_name"
     rds-database:
```

Comment:
Andy has suggested removing the hyphens for non-associations.  It has already caused me issues so once this successfully deploys I will rename in cloud-platform and in this service to ensure all syncs up ok.